docs/self-managed/operational-guides/update-guide/870-to-880.md

```diff
@@ -4,7 +4,177 @@ title: Update 8.7 to 8.8
 description: "Review which adjustments must be made to migrate from Camunda 8.7.x to Camunda 8.8.0."
 ---
 
-The following sections explain which adjustments must be made to migrate from Camunda 8.7.x to 8.8.x for each component.
+The following sections explain which adjustments must be made to migrate from Camunda 8.7.x to 8.8.x.
+
+## Camunda updating to 8.8
+
+The following Camunda 8.7 to 8.8 update guide covers changes related to the new harmonized data model, and the newly introduced [Camunda Exporter](/self-managed/zeebe-deployment/exporters/camunda-exporter.md/).
+Additional details about the introduced exporter and related changes you can find at the [announcement page](docs/reference/announcements/880.md).
+
+### Ensure the exporter/import backlog is small
+
+For data from 8.7 to be imported, the importers must run first and complete before the Camunda Exporter can start.
+
+The importer reads data from existing Zeebe Elasticsearch indices, and when it detects data from newer versions, it marks itself as complete. The Camunda Exporter then starts exporting only when the importer is done.
+
+:::important
+To reduce the potential of building a backlog, the user needs to make sure to drain the backlog queue as much as possible before the update.
+
+**Otherwise**, the backlog might increase, causing to not show up-to-date data in the web-applications. Furthermore, Zeebe
+will be not able to clean up data, causing to run Out-of-Disk space in worst case. This will produce down-time.
+:::
+
+The backlog can be validated via the following Prometheus metrics:
+
+- `operate_import_time_seconds_sum`
+- `operate_import_time_seconds_count`
+
+For example, the following query shows the import latency. If the result is in a range of seconds (ideally below 10 seconds), it is fine to update.
+
+```promql
+sum(rate(operate_import_time_seconds_sum{namespace=~"$namespace", partition=~"$partition", pod=~"$pod"}[$__rate_interval]))
+/ sum(rate(operate_import_time_seconds_count{namespace=~"$namespace", partition=~"$partition", pod=~"$pod"}[$__rate_interval]))
+```
+
+### Data migration
```

Comment:
@panagiotisgts I described a bit more the Migration application and the configs, could you please check this for correctness.